plugins/discovery/config.go

```diff
@@ -98,7 +98,7 @@ func (c *Config) validateAndInjectDefaults(services []string, confKeys map[strin
 	if c.Signing != nil {
 		err := c.Signing.ValidateAndInjectDefaults(copy)
 		if err != nil {
-			return fmt.Errorf("invalid configuration for discovery service %q: %s", *c.Name, err.Error())
+			return fmt.Errorf("invalid configuration for discovery service: %s", err.Error())
 		}
 	} else {
```

Comment:
From checking the diff, I couldn't tell if we had test coverage for this... Do we?

Reply:
We do, but only assert that any error is returned. (TestConfigValidation)

Reply:
Thanks!